spec-configs/props.conf.js

```diff
@@ -3,24 +3,28 @@ const path = require("path");
 const pkg = require(path.resolve(__dirname, "../package.json")); // eslint-disable-line
 
 module.exports = {
-  glob: ["./src/object-properties.js"],
-  package: path.resolve(__dirname, "../package.json"),
-  api: {
-    stability: "stable",
-    properties: {
-      "x-qlik-visibility": "public",
+  fromJsdoc: {
+    glob: ["./src/object-properties.js"],
+    package: path.resolve(__dirname, "../package.json"),
+    api: {
+      stability: "stable",
+      properties: {
+        "x-qlik-visibility": "public",
+      },
+      visibility: "public",
+      name: `${pkg.name}:properties`,
+      version: pkg.version,
+      description: "Action button generic object definition",
     },
-    visibility: "public",
-    name: `${pkg.name}:properties`,
-    version: pkg.version,
-    description: "Action button generic object definition",
-  },
-  output: {
-    file: path.resolve(__dirname, "../api-specifications/properties.json"),
-  },
-  parse: {
-    types: {
-      "qae.GenericObjectProperties": {},
+    output: {
+      file: path.resolve(__dirname, "../api-specifications/properties.json"),
+    },
+    parse: {
+      types: {
+        GenericObjectProperties: {
```

Comment:
```suggestion
        EngineAPI.GenericObjectProperties: {
```

Reply:
+ update in the props file and run it again

Reply:
But the we should also reference it like that here https://github.com/qlik-oss/sn-action-button/pull/428

Reply:
Also how wordcloud actually references it https://github.com/qlik-oss/sn-word-cloud/blob/main/api-spec/props.conf.js#L26